demo/api/src/products/generated/product.resolver.ts

```diff
@@ -32,19 +31,22 @@ export class ProductResolver {
 
     @Query(() => Product)
     @AffectedEntity(Product)
+    @RequiredPermission(["createProduct", "updateProduct", "deleteProduct"], { skipScopeCheck: true })
     async product(@Args("id", { type: () => ID }) id: string): Promise<Product> {
         const product = await this.repository.findOneOrFail(id);
         return product;
     }
 
     @Query(() => Product, { nullable: true })
+    @RequiredPermission(["createProduct", "updateProduct", "deleteProduct"], { skipScopeCheck: true })
     async productBySlug(@Args("slug") slug: string): Promise<Product | null> {
```

Comment:
should there be an extra option to define a permission for entityBySlug? currently the same as for single-resolver is used.

Reply:
no, also use `read`